src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalCatchCheck.java

```diff
@@ -56,7 +56,77 @@
  * <pre>
  * &lt;module name=&quot;IllegalCatch&quot;/&gt;
  * </pre>
+ * <pre>
+ * try {
+ *     // do stuff
+ * } catch (Exception e) { // Violation
+ *
+ * }
+ *
+ * try {
+ *     // do stuff
+ * } catch (ArithmeticException e) {
+ *
+ * } catch (Exception e) { // Violation, one of the exception is declared to be illegal
+ *                           and order of multi-catch blocks doesn't matter
```

Comment:
`and order of multi-catch blocks doesn't matter` it is not multicatch, just catch blocks